test/bin/stress_testing.sh

```diff
@@ -25,12 +25,46 @@ function run {
 
   if [ "${SSH_HOSTNAME:-}" ]; then
     # shellcheck disable=SC2086
-    ssh -o LogLevel=ERROR ${SSH_PORT_PARAM:-} ${SSH_PKEY_PARAM:-} ${SSH_USER_PARAM:-}${SSH_HOSTNAME} "$@"
+    ssh -o StrictHostKeyChecking=no -o LogLevel=ERROR ${SSH_PORT_PARAM:-} ${SSH_PKEY_PARAM:-} ${SSH_USER_PARAM:-}${SSH_HOSTNAME} "$@"
   else
     "$@"
   fi
 }
 
+# install dependecies
+function install_kernel-modules-extra {
+  if ( ! run rpm-ostree status &> /dev/null); then
```

Comment:
Nit: The () brackets are not usually used by us when checking command statuses in `if`.
Can we remove those in this script as well to adhere to the common style?

Reply:
thanks, round brackets removed